### PR TITLE
ui: fix build issue on arm/Mac

### PR DESCRIPTION
**Pull request by rohityadavcloud**
- Fixes UI build issue on M1/Mac

This is only useful if we're sure that node-sass isn't used by UI during development. The node-sass isn't a build dependency but only dev dependency.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### How Has This Been Tested?

- Run npm install and npm run serve which got no error
- Based on https://www.priestch.com/replace-node-sass-with-dart-sass-in-vue-cli3-based-project/ looks like we're not using sass at all in our vue.config or other places
- Not a sass/css expert but seems this doesn't cause any regression and most likely an unnecessary dependency

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
@blueorangutan ui

**Comment by rohityadavcloud** [comment]
Hi @svenvogel we may need a UI/css/sass expert to advise, based on my testing this doesn't cause any regression. Could you or any of your colleagues help advise as well? Thanks.

File: ui/package.json [hunk 1]
```diff
@@ -89,7 +89,6 @@
     "eslint-plugin-vue": "^7.0.0",
     "less": "^3.0.4",
     "less-loader": "^5.0.0",
-    "node-sass": "^4.0.0",
     "sass": "^1.49.9",
```
Comment (rohityadavcloud) [inline ui/package.json:92]: We're using the dart-sass already (see https://github.com/sass/node-sass#node-sass)

**Review by shwstppr — APPROVED** [review]
LGTM. node-sass has been deprecated and `sass` which is already used seems more popular

**Review by DaanHoogland — APPROVED** [review]

**Comment by DaanHoogland** [comment]
the qa version of this does not allow login :(

**Review by nvazquez — APPROVED** [review]
LGTM - I use M1 as well but was able to run the UI without this change. Have tested this PR and also works

**Review by harikrishna-patnala — APPROVED** [review]
LGTM, Tested.

Got the same issue before the fix and it is working fine after applying the fix. Thanks @rohityadavcloud

**Comment by weizhouapache** [comment]
code lgtm

tested on ubuntu 22, it is not broken :smile:

**Comment by rohityadavcloud** [comment]
@DaanHoogland works for me, could be caching issue - can you try in incognito mode?

**Comment by DaanHoogland** [comment]
> @DaanHoogland works for me, could be caching issue - can you try in incognito mode?

good call @rohityadavcloud , login works

**Pull request merged**